packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -197,12 +197,20 @@ export function encodeKeyHash(value: string) {
  * @throws {@link ValueConversionError}
  */
 export const hex2buf = (hex: string): Uint8Array => {
-  const match = hex.match(/[\da-f]{2}/gi);
-  if (match) {
-    return new Uint8Array(match.map((h) => parseInt(h, 16)));
-  } else {
+  if ((hex.length & 1) !== 0) {
```

Comment:
```suggestion
  if ((hex.length % 2) !== 0) {
```
It's more familiar to js developers

Reply:
Looks good